templates/blocks/block--system-branding-block.html.twig

```diff
@@ -6,12 +6,30 @@
  * @see ./core/modules/system/templates/block--system-branding-block.html.twig
  */
 #}
+{% set _header_link = {
+  url: url('<front>'),
```

Comment:
I usually prefer to defer this kind of code to the preprocess, but not a requirement.